### PR TITLE
Saros/E 16.0.0 release notes

### PR DESCRIPTION
**Pull request by m273d15**

### CODE REVIEW

File: docs/releases/saros-e_16.0.0.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+:tada: We are happy to announce **Saros for Eclipse 16.0.0** :tada:
+
+The new Saros for Eclipse 16.0.0 contains multiple [bug fixes](#bug-fixes)
+and [improvements](#improvements).
+
+## Installation
+Saros for Eclipse 16.0.0 requires
+ - `JDK 8` or newer
```
Comment (tobous) [inline docs/releases/saros-e_16.0.0.md:12]: Specify supported Eclipse versions?

**Comment by m273d15** [comment]
Thanks, I will add the badge stuff 👍

File: docs/releases/saros-e_16.0.0.md (hunk 1 above)
Comment (m273d15) [reply docs/releases/saros-e_16.0.0.md:12]: This is a good question. The oldest version listed in the [eclipse marketplace](https://marketplace.eclipse.org/content/saros-distributed-collaborative-editing-and-pair-programming#group-details) is " Eclipse Neon", but even if we switched to Neon as the development environment, we should be compatible with 4.* (probably also 3.* but this is nothing I would like to test).
We have to test this and also adjust the marketplace entry to >= 4.

Comment (m273d15) [reply docs/releases/saros-e_16.0.0.md:12]: Now I remember. We are compatible with >= 4.6 (Neon). Neon was the [first eclipse with JDK 8 runtime support](https://wiki.eclipse.org/Eclipse/Installation).

**Comment by m273d15** [comment]
Steps before merging:
- [X] Cherry pick: [version commit](https://github.com/saros-project/saros/commit/282d49e3084a6b287de95329186a59643d0a8e36), [empty selection fix](https://github.com/saros-project/saros/commit/24a655e6a507f393a3e2e889d9f9b3c331d665f7)
- [ ] Create dropin, create update-site artifacts
- [ ] Perform smoke tests with both artifacts
- [ ] Push update-site, upload dropin
- [ ] Publish GitHub release

**Comment by m273d15** [comment]
@stefaus @tobous We have to increase the Core's version number. Otherwise, the eclipse update mechanism will create an installation with the core-0.1.0 (from the 15.0.0) version and the saros.eclipse-16.0.0.
I would use core-0.2.0. Any objections?

File: README.md [hunk 2]
```diff
@@ -3,9 +3,10 @@
 [![STF Tests](https://github.com/saros-project/saros/workflows/STF%20Tests/badge.svg?branch=master&event=push)](https://github.com/saros-project/saros/actions?query=workflow%3A%22STF+Tests%22+branch%3Amaster)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/297b67607a5f4b5b8d00d0446615849b)](https://www.codacy.com/manual/Saros/saros)
 [![Gitter](https://img.shields.io/gitter/room/saros-project/saros?color=%2346BC99&logo=gitter)](https://gitter.im/saros-project/saros)
-[![Release Saros/E](https://img.shields.io/badge/Saros%2FE-15.0.0-blue?logo=eclipse)](https://github.com/saros-project/saros/releases/tag/saros-eclipse-15.0.0)
+[![Release Saros/E](https://img.shields.io/badge/Saros%2FE-16.0.0-blue?logo=eclipse)](https://github.com/saros-project/saros/releases/tag/saros-eclipse-15.0.0)
```
Comment (tobous) [inline README.md:6]: The first URL defines the badge (`Saros%2FE-16.0.0-blue?logo=eclipse` defines the text, color, and icon of the badge), the second is the link the badge points to. You have to update both.

```suggestion
[![Release Saros/E](https://img.shields.io/badge/Saros%2FE-16.0.0-blue?logo=eclipse)](https://github.com/saros-project/saros/releases/tag/saros-eclipse-16.0.0)
```

**Comment by tobous** [comment]
> @stefaus @tobous We have to increase the Core's version number. Otherwise, the eclipse update mechanism will create an installation with the core-0.1.0 (from the 15.0.0) version and the saros.eclipse-16.0.0.
> I would use core-0.2.0. Any objections?

No, sounds good to me. How about updating the Saros/E release instructions to mention this step?

File: docs/releases/saros-e_16.0.0.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Beside multiple internal reworks, [bug fixes](#bug-fixes) and [improvements](#improvements) we also
+[removed our Whiteboard](https://www.saros-project.org/contribute/deprecated/whiteboard.html).
+It was to brittle and does not fit into our plan to support further Editors/IDEs in the future.
+
+### Improvements
+* The `.git` directory is now ignored by default during a session ([#922](https://github.com/saros-project/saros/issues/922)).
+* Add Contacts Context Menu Item is enabled for Offline Contacts ([#19](https://github.com/saros-project/saros/issues/19))
+* Choose only one "XMPP ID" vs "Jabber ID" ([#22](https://github.com/saros-project/saros/issues/22))
+* Reusing existing project requires cumbersome manual selection ([#66](https://github.com/saros-project/saros/issues/66))
```
Comment (tobous) [inline docs/releases/saros-e_16.0.0.md:32]: This text seems to describe the previous behavior that was "fixed"/improved. This is kind of confusing as it is not directly apparent that these issues have been resolved.

I would suggest rewording it to make it more clear either that this was a problem that has been fixed or what the new behavior is.

The same also applies to the next section `Bug Fixes`. I would suggest changing the wording to show that the issues were resolved (e.g. by using past tense). Alternatively, you could rename the section to `Fixed Bugs`.

Comment (m273d15) [reply docs/releases/saros-e_16.0.0.md:32]: I will change there wording and will rename it to fixed bugs :+1:

File: docs/releases/saros-e_16.0.0.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Changes
+
+Beside multiple internal reworks, [bug fixes](#bug-fixes) and [improvements](#improvements) we also
+[removed our Whiteboard](https://www.saros-project.org/contribute/deprecated/whiteboard.html).
+It was to brittle and does not fit into our plan to support further Editors/IDEs in the future.
+
+### Improvements
+* The `.git` directory is now ignored by default during a session ([#922](https://github.com/saros-project/saros/issues/922)).
+* Add Contacts Context Menu Item is enabled for Offline Contacts ([#19](https://github.com/saros-project/saros/issues/19))
```
Comment (tobous) [inline docs/releases/saros-e_16.0.0.md:30]: This entry and the next entry need adjusting as well in my opinion. (The initial review comment was addressing the range from line 30 to line 32. This might not have been apparent.)

File: docs/contribute/processes/release.md [hunk 5]
```diff
@@ -18,6 +18,10 @@ Please make sure before creating the artifacts that **all unit and stf tests are
 * Update Site
 * Drop-in
 
+#### Prerequisite
+Make sure to increase the version number of the `saros.eclipse` **and** `saros.core` osgi bundle.
```
Comment (tobous) [inline docs/contribute/processes/release.md:22]: As a suggestion:

```suggestion
If there were any changes to the Saros core since the last release, make sure to also increase the version number of the `saros.core` osgi bundle.
```
